src/libsyntax/parse/parser.rs

```diff
@@ -5345,11 +5342,13 @@ impl<'a> Parser<'a> {
         ).emit();
     }
 
-    // Parse bounds of a type parameter `BOUND + BOUND + BOUND`, possibly with trailing `+`.
-    // BOUND = TY_BOUND | LT_BOUND
-    // LT_BOUND = LIFETIME (e.g., `'a`)
-    // TY_BOUND = TY_BOUND_NOPAREN | (TY_BOUND_NOPAREN)
-    // TY_BOUND_NOPAREN = [?] [for<LT_PARAM_DEFS>] SIMPLE_PATH (e.g., `?for<'a: 'b> m::Trait<'a>`)
+    /// Parses bounds of a type parameter `BOUND + BOUND + ...`, possibly with trailing `+`.
```

Comment:
Why are these suddenly doc comments on a private function when you made doc comments on private functions into normal comments elsewhere?

Reply:
Yeah, fair point. I noticed that doc comments were used a lot on private functions elsewhere in the codebase, so I probably got confused. Would be nice to get a "ruling" on this from someone like @nikomatsakis, @alexcrichton, or @steveklabnik.

Reply:
Okay, I resolved this in favour of using doc comments even on private methods for now.